.github/ISSUE_TEMPLATE/normative-change.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Normative Change
+about: If you are submitting a bug or a feature request that will require a normative
```

Comment:
about: This is a proposed feature that probably requires a specification change.